csrc/ir/internal_base_nodes.h

```diff
@@ -440,6 +440,12 @@ class TensorDomain : public Val {
       std::vector<IterDomain*> root_domain,
       std::vector<std::optional<bool>> contiguity = {});
 
+  TensorDomain(
+      IrBuilderPasskey,
+      std::vector<IterDomain*> root_domain,
+      std::vector<int64_t> stride_order,
```

Comment:
Add a comment explaining what `stride_order` means. E.g. it must be a permutation of [0 .. root_domain.size()-1]. It goes from major to minor (?), etc.

Reply:
Turns out we do have a comment for that. added a note to the comment.